src/Kanvas/AccessControlList/Repositories/ModulesRepositories.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\AccessControlList\Repositories;
+
+use Kanvas\Enums\ModuleEnum;
+use Kanvas\Apps\Models\Apps;
+use Kanvas\Companies\Models\Companies;
+use Kanvas\Users\Models\Users;
+use Kanvas\Inventory\Products\Models\Products;
+use Kanvas\Inventory\Products\Models\ProductsTypes;
+use Kanvas\Inventory\Regions\Models\Regions;
+use Kanvas\Inventory\Warehouses\Models\Warehouses;
+
+class ModulesRepositories
+{
+    public static function getAbilitiesByModule(): array
```

Comment:
@FredPeal thi s more a template than a repository , or enum, I wouldn't lvea this class in the repository folder